setup.py

```diff
@@ -9,9 +9,9 @@
     keywords=["time series", "forecasting"],
     version="0.7.1",
     install_requires=[
-        "pandas",
+        "pandas >= 1.4.0",
         "statsmodels",
-        "sklearn",
+        "scikit-learn",
```

Comment:
Did this branch happen before I made https://github.com/SeitaBV/timetomodel/commit/ce360560e6af800f9a5fd4bd3fe4c54a6dd12df8?